src/sidebar/services/view-filter.js

```diff
@@ -100,9 +100,9 @@ function viewFilter(unicode) {
     quote: {
       autofalse: ann => (ann.references || []).length > 0,
       value(annotation) {
-        if (!annotation.target) {
-          // FIXME: All annotations *must* have a target, so this check should
-          // not be required.
+        if (annotation.target.length === 0) {
+          // Annotations with no targets are draft page note
+          // annotations -- just ignore them.
```

Comment:
Looking at the coverage report (open `coverage/index.html` and navigate to `sidebar > services > view-filter.js`) the body of this `if` is never called because of the earlier check for whether an annotation has an id.

I had a look in "h" and the backend will currently always produce an annotation with a `target` array that has [exactly one entry](https://github.com/hypothesis/h/blob/0d4cd0c606bb62deb97a8dac2bded60efb95b517/h/presenters/annotation_base.py#L48). It could still be useful to have a test case in case that ever changes in future.